.eslintrc

```diff
@@ -0,0 +1,113 @@
+{
+  "env": {
+    "browser": true,
```

Comment:
Mostly to pick up on `document`, `window` etc. in the reporters, `lib/utils.js`, etc. We should probably remove it eventually though